tools/test_range.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+# Copyright 2019 OpenST Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# ----------------------------------------------------------------------------
+#
+# http://www.simpletoken.org/
+#
+# ----------------------------------------------------------------------------
+
+###
+# Runs tests sequentially for a range of decimal values.
+#
+# Example run with decimals 12 - 18 (inclusive):
+#
+# ```bash
+# npm run test:range -- 12 18
+# ```
+###
+
+echo ""
+
+if [ $# -ne 2 ];
```

Comment:
The acceptability criteria for this ticket indicate that entering 1 value should be acceptable.

Although, given your `seq` approach, I think including a statement in the echo documentation that providing the same value twice will achieve that result would be a simple way to satisfy the motivation behind this particular criterion.

Reply:
I will add that statement 👍

Reply:
The script will accept one and two arguments :+1